iOS/docs/kickoff/README.md

```diff
@@ -146,6 +146,25 @@ And commit them:
 
 To configure continuous integration follow the guide [CI configuration for iOS projects](./configure-ci/README.md)
 
+### Configure Crash logger
+
+The project integrates [Rollbar](https://github.com/rollbar/rollbar-ios) as default crash logger.
+
+In order to enable it, ask a technical leader to create a new group and a new iOS project.
+
+Once created, sign in to [Rollbar](https://rollbar.com), and go to your project "Settings"/"Access Tokens".
+
+Then copy into all the `.xcconfig` files the following content:
+
+```
+ROLLBAR_CLIENT_ACCESS_TOKEN = ...
+ROLLBAR_SERVER_ACCESS_TOKEN = ...
+```
+
+Where `ROLLBAR_CLIENT_ACCESS_TOKEN` is the token under "post_client_item", and `ROLLBAR_SERVER_ACCESS_TOKEN` is the token under "post_server_item".
+
+Once the `.xcconfig` files are configured (make sure they are ignored, if they are shown as changes to be committed, then do `git rm --cached ./BaseProject/ConfigurationFiles/*` to ignore this changes) just uncomment the call to `RollbarService().initialize()` in `AppDelegate.swift`.
```

Comment:
I think the `git rm --cached` will delete the files. Maybe they should use `git update-index --assume-unchanged <file>` ? I really never could make it work as I wanted.